src/main/java/com/example/tournamentMaker/tournament/result/ResultService.java

```diff
@@ -137,26 +138,26 @@ private MatchResult getOpposingTeamResult(MatchResult result) {
         }
     }
 
-    private void updateRecentResult(MatchResult result, List<MatchResult> recentResult) {
+    public void updateRecentResult(MatchResult result, List<MatchResult> recentResult) {
         recentResult.add(0, result);
         if (recentResult.size() > Constans.COLLECTED_MATCH_RESULTS_NUMBER) {
             recentResult.remove(Constans.COLLECTED_MATCH_RESULTS_NUMBER);
         }
     }
 
-    private MatchResult getHostResult(FootballResultRequest footballResultRequest) {
-        boolean draw = Objects.equals(footballResultRequest.getHostPoints(), footballResultRequest.getGuestPoints());
+    public MatchResult getHostResult(int hostPoints, int guestPoints) {
+        boolean draw = Objects.equals(hostPoints, guestPoints);
         if (draw) {
             return MatchResult.DRAW;
         }
-        boolean homeTeamWin = footballResultRequest.getHostPoints() > footballResultRequest.getGuestPoints();
+        boolean homeTeamWin = hostPoints > guestPoints;
         if (homeTeamWin) {
             return MatchResult.WIN;
         }
         return MatchResult.LOSE;
```

Comment:
ternary operator

Reply:
boolean homeTeamWin = hostPoints > guestPoints ? true : false

Tak jest bardziej czytelnie, czy jaki jest tego cel?

Reply:
cel taki ze mozna to zrobic w jednej linijce kodu a wiele na tym czytelnosc nie straci. 
Ze mozesz miec hostPoints > guestPoints ? WIN : LOSE